labelCloud/model/point_cloud.py

```diff
@@ -182,6 +185,35 @@ def from_file(
             write_buffer,
         )
 
+    def validate_segmentation_label(self) -> None:
+        unique_label_ids = set(np.unique(self.labels))  # type: ignore
+        unique_class_ids = set(c.id for c in LabelConfig().classes)
+        if not unique_class_ids.issuperset(unique_label_ids):
+            msg = QMessageBox()
+            msg.setWindowTitle("Invalid segmentation label")
+            msg.setText(
+                f"Segmentation labels {unique_label_ids} of `{self.path}` don't match with the label config {unique_class_ids}."
+            )
+            labels_to_replace = unique_label_ids.difference(unique_class_ids)
+            msg.setInformativeText(
```

Comment:
What is happening in the case the user clicks on "Cancel"? ... it keeps the old (unknown) label?

Reply:
Yes. It is done this way in case the user doesn't want to change their label files. However, if the user doesn't agree to replace, it will prompt every time it's checked.